pkg/auth/authorizer/interfaces.go

```diff
@@ -41,6 +41,9 @@ type Attributes interface {
 
 	// The kind of object, if a request is for a REST object.
 	GetResource() string
+
+	// In case a special endpoint like /api, /healthz has been called.
+	GetNonResourcePath() string
```

Comment:
document that either this or GetNamespace()/GetResource() return results, never both